### DISCUSSION TITLE
Using Bootstrap Modals in asp.net application. Move to new screen?

### DISCUSSION THREAD

**Discussion by SapphireGirl** [twbs/bootstrap#39236] [Q&A]
Can we configure a bootstrap modal so the user is able to move the modal to a second screen from the web app?

This is a .NET Core web application.

Our users would like to see the web app on one screen while viewing the modal on a different screen (monitor)

Is this even possible with Bootstrap Modals?

Thank you

**Comment by picobots** [comment]
@SapphireGirl This is impossible with Bootstrap (or any web framework for that matter). Bootstrap modals are just HTML DOM elements that look like modal windows, they're not actual windows. You can't drag individual HTML elements out of a web browser onto a second monitor. (One weird exception: You could technically make a single browser window extra wide so that it spans two monitors and then I suppose you could achieve this, but that would be a wildly unusual approach.)

In practice, the only way to achieve your second monitor goal would be moving the content in question to a new browser tab or popup window, and then the user can move that browser tab/window to a second monitor.
